skimage/filters/__init__.py

```diff
@@ -53,6 +54,8 @@
            'rank_order',
            'gabor_kernel',
            'gabor',
+           'frangi_filter',
+           'hessian_filter',
```

Comment:
These should be renamed just "frangi" and "hessian" for consistency (see above, "gaussian", "gabor", "roberts", etc.